cmd/world/cardinal/dev.go

```diff
@@ -65,62 +69,36 @@ var devCmd = &cobra.Command{
 			isRedisRunning = true
 		}
 
-		// Run Cardinal
-		cardinalExecCmd, err := runCardinal()
-		if err != nil {
-			return err
-		}
-
 		// Create a channel to receive termination signals
 		signalCh := make(chan os.Signal, 1)
-		signal.Notify(signalCh, os.Interrupt, syscall.SIGTERM)
+		signal.Notify(signalCh, syscall.SIGINT, syscall.SIGTERM)
 
-		// Create a channel to receive errors from the command
-		cmdErr := make(chan error, 1)
+		// Run Cardinal Preparation
+		err = runCardinalPrep()
+		if err != nil {
+			return err
+		}
 
 		fmt.Println("Starting Cardinal...")
+		go func(stopChan <-chan struct{}) {
```

Comment:
nit: stopChan seems to be unused. Wouldn't this be the same:

go runner.Start()

Reply:
Hmm after i test it again, yup that's right, its unused.
My purpose is, i want to stop the go routine process when i'm closing the channel (for unit test purpose).

i'll be back with another method.

Reply:
Updated